pkgs/development/python-modules/bx-python/default.nix

```
@@ -0,0 +1,37 @@
+{ lib, fetchFromGitHub, buildPythonPackage, numpy, cython, zlib, six, python-lzo
+, nose }:
+
+buildPythonPackage rec {
+  pname = "bx-python";
+  version = "0.8.6";
+
```

only python2 interpreter is python2.7
```suggestion
  disabled = isPy27;

```